identity/README.md

```diff
@@ -234,6 +234,13 @@ message Capability {
       // plugin can invoke RPCs that require access to the storage system,
       // similar to the CSI Controller (provisioner).
       NETWORK_FENCE = 1;
+
+      // GET_CLIENTS_TO_FENCE indicates that the CSI-driver provides RPCs for a
+      // GET_CLIENTS_TO_FENCE operation to get the clients to fence.
+      // The presence of this capability determines whether the CSI-Addons CO
+      // plugin can invoke RPCs that require access to the storage system,
+      // similar to the CSI Controller (provisioner).
```

Comment:
Thanks for fixing the typos, however there is now _clinets_ in the commit description :laughing:

Reply:
Addressed now 🤦🏻